src/pyrokinetics/templates.py

```diff
@@ -7,11 +7,13 @@
 gk_cgyro_template = template_dir / "input.cgyro"
 gk_gene_template = template_dir / "input.gene"
 gk_tglf_template = template_dir / "input.tglf"
+gk_stella_template = template_dir / "input.stella"
 gk_templates = {
     "GS2": gk_gs2_template,
     "CGYRO": gk_cgyro_template,
     "GENE": gk_gene_template,
     "TGLF": gk_tglf_template,
+    "stella": gk_stella_template,
```

Comment:
To best consistent this should be uppercase

```suggestion
    "STELLA": gk_stella_template,
```

Reply:
OK, fine for me. The stella name convention is to use lower case for the code name, but I am happy either way here so long as it's obvious (which it might be from your documentation, once I update it).

Reply:
Hopefully the last commits address this concern @bpatel2107?